lib/membrane/rtp/twcc_receiver.ex

```diff
@@ -0,0 +1,155 @@
+defmodule Membrane.RTP.TWCCReceiver do
+  @moduledoc """
+  The module defines an element responsible for recording transport-wide statistics of incoming packets.
+  """
+  use Membrane.Filter
+
+  alias Membrane.{RTP, RTCPEvent, Time}
+  alias Membrane.RTP.Header
+  alias Membrane.RTCP.TransportFeedbackPacket
+  alias __MODULE__.PacketInfoStore
+
+  @feedback_count_limit 256
+
+  def_input_pad :input, demand_unit: :buffers, caps: RTP, availability: :on_request
+  def_output_pad :output, caps: RTP, availability: :on_request
+
+  def_options twcc_id: [
+                spec: 1..14,
+                description: "ID of TWCC header extension."
+              ],
+              report_interval: [
+                spec: Membrane.Time.t(),
+                default: Membrane.Time.milliseconds(250),
+                description: "How often to generate feedback packets."
+              ],
+              sender_ssrc: [
+                spec: RTP.ssrc_t(),
+                default: nil,
+                description:
+                  "Sender SSRC for generated feedback packets (will be supplied by `RTP.SessionBin`)."
+              ]
+
+  defmodule State do
+    @moduledoc false
+    alias Membrane.RTP.TWCCReceiver.PacketInfoStore
+
+    @type t :: %__MODULE__{
```

Comment:
I don't think that this typespec is correct (unless RTP.ssrc_t() specifies that it can be nil). Both of your `sender/media` ssrcs can be nil.

Reply:
Fixed